sales_team_security_sale/readme/DESCRIPTION.rst

```diff
@@ -1 +1,7 @@
 Glue module between sales_team_security and sale.
+
+This module adds a new type of "Sale" group called "User: Team Documents Only",
```

Comment:
This is not correct. This module doesn't add this group. It's added by the dependent module. This module adds the permissions to sales documents according such group.

Reply:
@cesar-tecnativa @LudLaf  please finish this

Reply:
Fixed.
@pedrobaeza please confirm. Thanks

Reply:
OK.